### DISCUSSION TITLE
Receiving many messages with no message body "No Messages"

### DISCUSSION THREAD

**Discussion by phyokyaw** [chatwoot/chatwoot#1951] [Q&A]
Dear all,

We are new to chatwoot. At our public server we start to receive messages with no body. Is this a bot attack, any clues on where to start tracing? We haven't done much investigation.

Thanks in advance,
PK

![no_message](https://user-images.githubusercontent.com/6470239/111857098-a9e12300-895d-11eb-922f-501b275a1cf4.png)

**Comment by pranavrajs** [comment]
@phyokyaw What is the version of Chatwoot you are using? There was a bug that caused this. Please use 1.13.1 or later.

**Reply by phyokyaw** [reply]
Many thanks for the reply. We are using 1.12.2. So we will update for sure! 

Is there issue reference for this bug please? Just a hint to search may be, I tried but can't find it.

Many thanks,
PK

**Reply by pranavrajs** [reply]
https://github.com/chatwoot/chatwoot/issues/1480
